Clicker/Views/Settings/SettingsLogOutButtonCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  SettingsLogOutButtonCell.swift
+//  Clicker
+//
+//  Created by Lucy Xu on 3/23/19.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol LogoutButtonCellDelegate: class {
+    func logOutButtonCellWasTapped()
+}
+
+class SettingsLogOutButtonCell: UICollectionViewCell {
+    
+    // MARK: Data vars
+    weak var delegate: LogoutButtonCellDelegate?
+    
+    // MARK: Views and VC's
+    var logOutButton: UIButton!
+    var linkURL: URL!
+    
+    // MARK: Layout constants
+    let standardInset = 18
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupViews()
+        setupConstraints()
+    }
+    
+    func setupViews() {
+        logOutButton = UIButton()
+        logOutButton.setTitleColor(UIColor.clickerGreen0, for: .normal)
+        logOutButton.titleLabel?.textAlignment = .left
+        logOutButton.titleLabel?.font = UIFont._18MediumFont
+        logOutButton.contentHorizontalAlignment = .left
+        logOutButton.addTarget(self, action: #selector(logOutClicked), for: .touchUpInside)
+        contentView.addSubview(logOutButton)
+    }
+    
+    func setupConstraints() {
+        logOutButton.snp.makeConstraints { make in
+            make.left.equalToSuperview().offset(standardInset)
+            make.right.equalToSuperview()
+            make.centerY.equalToSuperview()
```

Comment:
can you chain these to be `make.right.centerY.equalToSuperview()`

Reply:
Addressed in most recent pull request!